front/commonitiltask.form.php

```diff
@@ -113,7 +113,7 @@
         4,
         "tracking",
         //TRANS: %s is the user login
-        sprintf(__('%s updates a task'), $_SESSION["glpiname"])
+        sprintf(__('%s unplan a task'), $_SESSION["glpiname"])
```

Comment:
```suggestion
        sprintf(__('%s unplans a task'), $_SESSION["glpiname"])
```
 I'm not entirely sure :)